test/controllers/api/v2/smart_class_parameters_controller_test.rb

```diff
@@ -192,4 +192,40 @@ class Api::V2::SmartClassParametersControllerTest < ActionController::TestCase
     get :show, {:id => lookup_keys(:five).id}, set_session_user.merge(:user => users(:one).id)
     assert_response :success
   end
+
+  test "should not accept boolean that is not sent as string" do
```

Comment:
This code is duplicated because you're testing a concern, can you test it in `/test/controllers/shared` or something similar so that the code isn't basically identical in 2 places?